remote-trait-object-tests/src/mod_ping/traits.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright 2020 Kodebox, Inc.
+// This file is part of CodeChain.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pub trait PingInterface {
```

Comment:
I think just `Ping` sounds better because we all know that this is an `Interface`.

Reply:
The trait used in remote-trait-object is special. I want to make it explicit by adding "Interface" in its name.
Do you have any idea about making the information "this trait is used by RemoteTraitObject" explicit without the name?

Reply:
@majecty Then let's keep it until we use `Ping : ServiceHandler`